aspnetcore/tutorials/first-mvc-app-xplat/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Building your first ASP.NET Core MVC app with Visual Studio | Microsoft Docs
+author: rick-anderson
+description: 
+keywords: ASP.NET Core,
+ms.author: riande
+manager: wpickett
+ms.date: 10/14/2016
+ms.topic: article
+ms.assetid: 959ff4ca-be2d-1638-ac04-789e7f7146ff
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: tutorials/first-mvc-app-xplat/index
+---
+# Building your first ASP.NET Core MVC app with Visual Studio
+
+This series of tutorials will teach you the basics of building an ASP.NET Core MVC web app using Visual Studio.
```

Comment:
If you're going to publish in this incomplete state, it would be worth noting here and in tutorial 1 that it's a work in progress and more tutorials are on the way.